diff -ruN dc400b24-src./source/makefile.lnx dc400b24-src/source/makefile.lnx
--- dc400b24-src./source/makefile.lnx	Wed May 29 11:30:02 2002
+++ dc400b24-src/source/makefile.lnx	Fri Jan 17 01:20:46 2003
@@ -20,8 +20,8 @@
 
 LDFLAGS = -static -L/usr/lib
 MCHMOD = 2755
-# INSTALLDIR = /usr/games
-INSTALLDIR = /opt/crawl/bin
+INSTALLDIR = $(DESTDIR)/usr/games
+#INSTALLDIR = /opt/crawl/bin
 LIB = -lncurses
 
 # Include for Linux
@@ -31,7 +31,7 @@
 
 install: $(GAME)
 	$(COPY) $(GAME) ${INSTALLDIR}
-	chmod ${MCHMOD} ${INSTALLDIR}/$(GAME)
+	/bin/true ${MCHMOD} ${INSTALLDIR}/$(GAME)
 
 clean:
 	$(DELETE) *.o
@@ -50,7 +50,7 @@
 
 $(GAME): $(OBJECTS)
 	${CXX} ${LDFLAGS} $(INCLUDES) $(CFLAGS) $(OBJECTS) -o $(GAME) $(LIB)
-	chmod ${MCHMOD} $(GAME)
+	/bin/true ${MCHMOD} $(GAME)
 
 debug: $(OBJECTS)
 	${CXX} ${LDFLAGS} $(INCLUDES) $(CFLAGS) $(OBJECTS) -o $(GAME) $(LIB)
@@ -59,7 +59,7 @@
 	${CXX} -g -p ${LDFLAGS} $(INCLUDES) $(CFLAGS) $(OBJECTS) -o $(GAME) $(LIB)
 
 .cc.o:
-	${CXX} ${CFLAGS} -c $< ${INCLUDE}
+	${CXX} ${CFLAGS} -c $< ${INCLUDES}
 
 .h.cc:
 	touch $@
